### PR TITLE
css only dropdown

### PR DESCRIPTION
**Pull request by tedk13**
New blog post.

### CODE REVIEW

**Review by mrtrimble — COMMENTED** [review]
give 'em the ol' CSS razzle dazzle!

**Comment by tedk13** [comment]
@mrtrimble 
giphy-15

File: _posts/2020-12-11-css-only-dropdown-menu.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  <span>See the Pen <a href="https://codepen.io/PhiloBeddoe/pen/GRJBdEZ">
+  Simple Nav Dropdown with css transition</a> by Ted Krueger (<a href="https://codepen.io/PhiloBeddoe">@PhiloBeddoe</a>)
+  on <a href="https://codepen.io">CodePen</a>.</span>
+</p>
+<script async src="https://cpwebassets.codepen.io/assets/embed/ei.js"></script>
+<br>
+
+Notice in the example above, when you click Dropdown Action, the dropdown menu displays. The dropdown has a `max-height: 0;` by default. After the cliick, we add a class of `active` to the dropdown and change the `max-height` property. I set it to be `100vh` when active. Remember, this is just a `max-height` so by doing this I’m saying the dropdown is going to be at most the height of the window. From here you could add some overflow styles to ensure that users could scroll so they see all the content within the dropdown, or you could change the number. The important part is that the number does need to be specific. Hopefully, this is something we see in the future for CSS, but for now `max-height` it is.
```
Comment (hougasian) [inline _posts/2020-12-11-css-only-dropdown-menu.md:35]: is `cliick` Finnish?

Comment (tedk13) [reply _posts/2020-12-11-css-only-dropdown-menu.md:35]: Norwegian.

**Pull request merged**